src/remote-config/remote-config-api.ts

```diff
@@ -213,6 +218,123 @@ export interface PercentCondition {
   microPercentRange?: MicroPercentRange;
 }
 
+/**
+ * Defines supported operators for custom signal conditions.
+ */
+export enum CustomSignalOperator {
+
+  /**
+   * A catchall error case.
+   */
+  UNKNOWN = 'UNKNOWN',
+
```

Comment:
Per offline discussion, these limitations are enforced on the management planes (when defining these conditions). They aren't as relevant here in the SDK where the value that's passed in is one single value. As such, we won't highlight too much here to prevent any confusion